front/components/assistant/conversation/ConversationViewer.tsx

```diff
@@ -71,6 +72,9 @@ import { Err, isContentFragmentType, isUserMessageType, Ok } from "@app/types";
 
 const DEFAULT_PAGE_LIMIT = 50;
 
+// A conversation must be unread and older than that to enable the suggestion of enabling notifications.
+const DELAY_BEFORE_SUGGESTING_PUSH_NOTIFICATION_ACTIVATION = 60 * 60 * 1000; // 1 hour
```

Comment:
```suggestion
const DELAY_BEFORE_SUGGESTING_PUSH_NOTIFICATION_ACTIVATION_MS = 60 * 60 * 1000; // 1 hour
```

I find it a good practice to have the unit in variables